pkg/runner/job_executor.go

```diff
@@ -95,21 +95,13 @@ func newJobExecutor(info jobInfo, sf stepFactory, rc *RunContext) common.Executo
 	}
 
 	postExecutor = postExecutor.Finally(func(ctx context.Context) error {
-		logger := common.Logger(ctx)
 		jobError := common.JobError(ctx)
-		if jobError != nil {
-			info.result("failure")
-			logger.WithField("jobResult", "failure").Infof("\U0001F3C1  Job failed")
-		} else {
-			err := info.stopContainer()(ctx)
-			if err != nil {
-				return err
-			}
-			info.result("success")
-			logger.WithField("jobResult", "success").Infof("\U0001F3C1  Job succeeded")
+		var err error
+		if rc.Config.AutoRemove || jobError == nil {
+			err = info.stopContainer()(ctx)
```

Comment:
This will fail if the job was canceled before (e.g. by os signal).
Please make sure that a fallback context is used in that case.

Reply:
> Do we want to add a forceCancel context as field to the context, so we can force quit act by pressing ctrl+c twice? This could be the parent context of the context with a timeout.

That was the case before. The `BackgroundContext` used as a fallback was cancelable as well. That would then cancel/stop the cleanup.

Reply:
I understood and I'm fine with that.
I just thought that it did work before, as a second cancelation was respected by act (if I remember correctly).